okta-jwt-sso/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+services:
+  metabase:
+    build: metabase/.
+    container_name: metabase_full_app_embedding_demo
+    hostname: metabase
+    ports:
+      - 3000:3000
+    environment: 
+      - "MB_SITE_URL=http://localhost:3000"
+      - "MB_EMBEDDING_APP_ORIGIN=*"
+      - "MB_ENABLE_EMBEDDING=true"
```

Comment:
This is deprecated and will be removed in Metabase 53. Let's change it to the separate env vars for embedding.